README.adoc

```diff
@@ -219,30 +328,55 @@ SystemClient.java
 include::finish/inventory/src/main/java/io/openliberty/guides/inventory/client/SystemClient.java[]
 ----
 
-The changes introduced here use MicroProfile Config and CDI to inject the value of the environment variables [hotspot=system-app-username]`SYSTEM_APP_USERNAME` and [hotspot=system-app-password]`SYSTEM_APP_PASSWORD` into the [hotspot file=0]`SystemClient` class.
+The changes introduced here use MicroProfile Config and CDI to inject the value of the
+environment variables [hotspot=system-app-username]`SYSTEM_APP_USERNAME` and
+[hotspot=system-app-password]`SYSTEM_APP_PASSWORD` into the [hotspot file=0]`SystemClient` class.
 
 
 == Creating a ConfigMap and Secret
 
-There are several ways to configure an environment variable in a Docker container. You can set it directly in the [hotspot file=1]`Dockerfile` with the `ENV` command. You can also set it in your [hotspot file=0]`kubernetes.yaml` file by specifying a name and a value for the environment variable you want to set for a specific container. With these options in mind, you're going to use a ConfigMap and Secret to set these values. These are resources provided by Kubernetes that are used as a way to provide configuration values to your containers. A benefit is that they can be reused across many different containers, even if they all require different environment variables to be set with the same value.
+There are several ways to configure an environment variable in a Docker container.
+You can set it directly in the [hotspot file=1]`Dockerfile` with the `ENV` command.
+You can also set it in your [hotspot file=0]`kubernetes.yaml` file by specifying a
+name and a value for the environment variable you want to set for a specific container.
+With these options in mind, you're going to use a ConfigMap and Secret to set these values.
+These are resources provided by Kubernetes that are used as a way to provide configuration values to your containers.
```

Comment:
These are resources provided by Kubernetes that are used as a way to provide configuration values to your containers.
---->
These resources are provided by Kubernetes as a way to provide configuration values to your containers.